pages/wi/partner-listing.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+import { Grid, Cell } from 'styled-css-grid';
+import ContentSection from '../../components/shared/ContentSection';
+import LinkButton from '../../components/shared/LinkButton';
+import { below } from '../../utilities/breakpoint';
+
+const GET_PARTNERS = gql`
+  query getPartners {
+    partners {
+      id
+      year
+      partnershipLevel
+      companyName
+      companyLogo
+      heroImage
+      website
+    }
+  }
+`;
+
+const RobotImage = styled.img`
+  height: 500px;
```

Comment:
be great to convert this to `rem`

Reply:
looks like I missed some 'px' -> 'rem' clean up on listing...fixed.